articles/logic-apps/logic-apps-move-from-mabs.md

```diff
@@ -173,9 +173,9 @@ as well as XPath using the xpath workflow definition language expression.
 
 ### Run custom code
 
-BizTalk Services lets you [run custom code](https://msdn.microsoft.com/library/azure/dn232389.aspx) 
+BizTalk Services lets you [run custom code](/previous-versions/azure/dn232389(v=azure.100)) 
```

Comment:
Please use this URL:

https://docs.microsoft.com/previous-versions/azure/dn232389(v=azure.100)